dGame/dUtilities/SlashCommandHandler.cpp

```diff
@@ -1300,6 +1301,35 @@ void SlashCommandHandler::HandleChatCommand(const std::u16string& command, Entit
 		}
 	}
 
+	if ((chatCommand == "bubble") && entity->GetGMLevel() >= GAME_MASTER_LEVEL_DEVELOPER) {
+
+		auto controllablePhysicsComponent = entity->GetComponent<ControllablePhysicsComponent>();
+		if (!controllablePhysicsComponent) return;
+
+
+		if (args.size() == 0) {
+			controllablePhysicsComponent->DeactivateBubbleBuff();
+		} else{
+			uint32_t type_intermed;
+			if (!GeneralUtils::TryParse(args[0], type_intermed)) {
+				ChatPackets::SendSystemMessage(sysAddr, u"Invalid bubbleType.");
+				return;
+			}
+			bool floating = true;
+			if (args.size() == 2) {
+				uint32_t type_intermed2;
```

Comment:
Do booleans need an intermediate value?  
Following with our style guidelines these should be named typeIntermed as well.

Reply:
yeah, I'll fix that, wasn't going to include the command, but decided to last minute

Reply:
To clarify why I used an intermed for floating, it didn't work the way I expected so I just made it more verbose about it to be explict. 
Which I think is fine for a GM command to test this functionality